composeml/tests/test_label_maker.py

```diff
@@ -261,7 +341,7 @@ def test_search_offset_mix_7(transactions, total_spent_fn):
     given_labels = lm.search(
         transactions,
         num_examples_per_instance=float('inf'),
-        verbose=False,
+        verbose=True,
```

Comment:
is this test meant to test `verbose`?

Reply:
No, the `versbose` parameter is not needed in the test. I can remove it.